README.md

```diff
@@ -34,44 +34,53 @@ Beyond what is necessary to test the generator, the Cocina model classes are not
 
 ## Releasing
 
+### Step 0: Share intent to change the models
+
+Send a note to `#dlss-infra-chg-mgmt` on Slack to let people know what is changing and when.
+
+### Step 1: Cut the release
+
 The release process is much like any other gem. First bump the version in `lib/cocina/models/version.rb`, and commit the result. Then run:
 ```
 bundle exec rake release
 ```
 which pushes the gem to rubygems.org.  Next write up the release notes: https://github.com/sul-dlss/cocina-models/releases .
 
-Finally, you must release versions of [sdr-client](https://github.com/sul-dlss/sdr-client) and [dor-services-client](https://github.com/sul-dlss/dor-services-client/) pinned to this version because [Argo](https://github.com/sul-dlss/argo) depends on both of those. When [dor-services-app](https://github.com/sul-dlss/dor-services-app) is updated to use the new models (via the auto-update script), the clients must be updated at the same time or there is risk of models produced by dor-services-app not being acceptable to the clients.
```

Comment:
None of the text in the README was actually removed. It was just shifted around a bit to hopefully communicate the organization and order of information more clearly and more effectively.